ci/ray_ci/test_linux_tester_container.py

```diff
@@ -134,7 +140,13 @@ def _mock_run_tests_in_docker(
     def _mock_shard_tests(tests: List[str], workers: int, worker_id: int) -> List[str]:
         return chunk_into_n(tests, workers)[worker_id]
 
-    with mock.patch(
+    with tempfile.TemporaryDirectory() as tmpdir, mock.patch(
+        "ci.ray_ci.linux_tester_container.LinuxTesterContainer.get_artifact_mount",
+        return_value=("/tmp/artifacts", tmpdir),
```

Comment:
why does it need this mocking? now you can send in a real temp dir in testing I guess?